src/components/surveys/survey-design/question-edit/QuestionEditPhone.tsx

```diff
@@ -143,25 +144,28 @@ function Factory(args: {
 
     case 'FREE_TEXT':
       return (
-        <FakeInput
-          width={200}
-          height={20}
+        <Box
           sx={{
-            borderRight: 'none',
-            borderTop: 'none',
-            borderBottom: 'none',
-            background: 'transparent',
-          }}
-        />
-      ) //<FreeText step={args.step} onChange={args.onChange} />
+            fontFamily: latoFont,
+            fontSize: '14px',
+            fontStyle: 'italic',
+            fontWeight: '400',
+            textAlign: 'left',
+          }}>
+          {(args.step as Question).inputItem?.placeholder}
+        </Box>
+      ) // Mobile devices do not support more than 250 characters of free text.
```

Comment:
Could consider pulling this into its own component like the other subcontrols, e.g. something like TimeDuration.tsx or Numeric.tsx in src/components/surveys/survey-design/question-edit/phone-subcontrols

Reply:
Done